notebooks/Dataset Creation and Experiment Selection.ipynb

```diff
@@ -63,12 +61,12 @@
    "source": [
     "# API Endpoint and Credentials\n",
     "host_url = \"https://api.monai.ngc.nvidia.com\"\n",
-    "ngc_api_key = os.environ.get('MONAI_API_KEY')\n",
+    "ngc_api_key = os.environ.get(\"MONAI_API_KEY\", \"<YOUR_API_KEY>\")  # we recommend using environment variables for API keys, but you can also hardcode them here\n",
```

Comment:
Update here too and other auth sections.

Reply:
I will create separate issue

Reply:
Issue in https://github.com/NVIDIA/monai-cloud-api/issues/79